### DISCUSSION TITLE
what firmware versions on v2 and V3 still have BOA enabled

### DISCUSSION THREAD

**Discussion by eloquentvogon** [mrlt8/docker-wyze-bridge#512] [Q&A] [answered]
I have been digging around and I have not found a version that the "enable_boa=TRUE" will work on for the V2 or V3 cameras. Are there any versions available? TIA

**Comment by mrlt8** [comment, marked as answer]
I believe they started disabling BOA sometime in march, so you could try something before that?

**Reply by eloquentvogon** [reply]
I have rolled back to 4.9.8.860 on V2 and still no BOA. Was it ever available on the V2?

**Reply by mrlt8** [reply]
It _should_ work on the v2. Make sure you're connected over LAN mode and have a working SD card in the camera.

Can you post the `sdParm` from your camera? It should be visible in the WebUI

**Reply by eloquentvogon** [reply]
seems the SD card is not being seen by the wyze firmware, I can load it via SD but then the camera never sees it, SDPARM is 2 which looks like it means it cannot be seen. Going to retest on my pan cam. Thanks

'sdParm': {'status': '2', 'capacity': '0', 'free': '0'},

**Reply by eloquentvogon** [reply]
issue resolved, may have been a bad sd card or the V2 is not seeing it, either way, pancam on 4.10.8.860 shows the BOA url. thanks for the help
